pallets/did/src/did_details.rs

```diff
@@ -402,14 +430,39 @@ impl<T: Config> DidDetails<T> {
 		Ok(new_did_details)
 	}
 
+	// Creates a new DID entry from some [DidCreationDetails] and a given
+	// authentication key.
+	pub fn from_account(
+		submitter: AccountIdOf<T>,
```

Comment:
I think also here it would be more correct to name this `origin`, right? It might not be a tx submitter.

Reply:
The whole function isn't that great. I thing it should rather be called `new_with_...` and also it's not really that different to `new` anyways so i merged these two. [9b139f6](https://github.com/KILTprotocol/kilt-node/pull/551/commits/9b139f68f50eca7acd6420f57f028e30683cc75f)

The submitter here should be called `deposit_owner` IMO since that's what the accountId is used for in the end.

Reply:
Like it!